src/core/Cheatsheet.ts

```diff
@@ -111,10 +111,38 @@ export default class Cheatsheet {
       "defaults.json"
     );
 
-    await writeFile(defaultsPath, JSON.stringify(spokenFormInfo, null, "\t"));
+    const outputObject = produce(spokenFormInfo, (draft) => {
+      draft.sections = sortBy(draft.sections, "id");
+      draft.sections.forEach((section) => {
+        section.items = sortBy(section.items, "id");
+      });
+    });
```

Comment:
I think [immer](https://immerjs.github.io/immer/) is a nice way to handle immutable data structures.  Planning to start using it if no objections

thanks for the pointer @SimeonC

Reply:
👍 It does seem like it's more suited to our use case than immutable.js which seems to be the other choice here. I found an article comparing the two: https://blog.logrocket.com/immer-and-immutable-js-how-do-they-compare/

Reply:
That's what they do [in the docs](https://immerjs.github.io/immer/#with-immer) 🤷‍♂️